stripe-patrons-data/build.sbt

```diff
@@ -42,6 +44,13 @@ assembly / assemblyMergeStrategy := {
     oldStrategy(y)
 }
 
+// We also have to put the build number in the .jar, because AWS refuses to create a new lambda version if the jar is the same!
+resourceGenerators in Compile += Def.task {
+  val file = (resourceManaged ).value / "build.number"
+  IO.write(file, buildNumber)
+  Seq(file)
+}.taskValue
```

Comment:
I've had to add this additional task in the build.sbt because the deploy was failing if the contents of the jar didn't change between deploys - even though the filename was different! AWS is very strict.
This task generates a file with the build number, to ensure each jar is unique